src/lti/contrib/django/README.md

```diff
@@ -0,0 +1,28 @@
+## Install ##
+
+`pip install git+https://github.com/ksofa2/lti.git@django`
```

Comment:
And I'd prefer if this was reStructuredText instead of Markdown, to match what I do elsewhere.